### ISSUE TITLE
Can you bump three.js peerDependency version number to 0.149.0?

### ISSUE THREAD

**Issue by gouldingken** [pmndrs/postprocessing#450]
Is there a reason that v 149.0 is explicitly not supported?

>= 0.125.0 < 0.149.0

This is currently causing issues when installing @react-three/postprocessing 2.7.0 via pnpm

```
└─┬ @react-three/postprocessing 2.7.0
  └─┬ postprocessing 6.29.3
    └── ✕ unmet peer three@">= 0.125.0 < 0.149.0": found 0.149.0
```

**Comment by gouldingken** [comment]
Ok, apologies. I now see that v 0.149.0 does NOT work with this package, so the peer dependency versioning is correct. I'll close this and add an issue to @react-three/postprocessing

**Comment by vanruesc** [comment]
This issue does belong here and there's a fix in #448 waiting to be released, but I currently don't have access to my PC because I'm moving.
